### PR TITLE
Feat/handle ivv

### PR DESCRIPTION
**Pull request by GhislainJ**
* **What kind of changes does this PR introduce?** 
- [ ] Bug fix
- [ ] new features
- [ ] performance 
- [ ] docs update

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- [ ] Yes:
- [ ] No

* **Other information**:

### CODE REVIEW

File: dessia_common/workflow/blocks.py [hunk 1]
```diff
@@ -84,11 +85,11 @@ class InstantiateModel(Block):
...
     """
 
-    def __init__(self, model_class: Type, name: str = "", position: Tuple[float, float] = None):
+    def __init__(self, model_class: Type, name: str = "Instantiate Model", position:  Position = (0, 0)):
         self.model_class = model_class
         inputs = []
         inputs = set_inputs_from_function(self.model_class.__init__, inputs)
-        outputs = [TypedVariable(type_=self.model_class, name="Model")]
+        outputs = [Variable(type_=self.model_class, name="Model")]
         Block.__init__(self, inputs, outputs, name=name, position=position)
```
Comment (Thiefaine-dev) [inline dessia_common/workflow/blocks.py:93]: Same comment here

File: dessia_common/workflow/blocks.py [hunk 2]
```diff
@@ -164,6 +173,18 @@ def equivalent(self, other: 'ClassMethod'):
         same_method = self.method_type.name == other.method_type.name
         return Block.equivalent(self, other) and same_class and same_method
```
Comment (Thiefaine-dev) [inline dessia_common/workflow/blocks.py:174]: Same comment here

File: dessia_common/workflow/blocks.py [hunk 3]
```diff
@@ -1282,14 +1392,14 @@ class Archive(Block):
...
+    def __init__(self, number_exports: int = 1, filename: str = "archive", name: str = "Archive",
+                 position:  Position = (0, 0)):
         self.number_exports = number_exports
         self.filename = filename
         self.extension = "zip"
         self.text = False
         inputs = [Variable(name=f"Export {i}") for i in range(number_exports)]
-        inputs.append(TypedVariableWithDefaultValue(type_=str, default_value=filename, name="Filename"))
+        inputs.append(Variable(type_=str, default_value=filename, name="Filename"))
         Block.__init__(self, inputs=inputs, outputs=[Variable(name="Archive")], name=name, position=position)
```
Comment (Thiefaine-dev) [inline dessia_common/workflow/blocks.py:1403]: A lot of init use `NameOfClass.__init__()`. Does it have a reason? If not it's preferable to use `super().__init__`

**Review by Thiefaine-dev — CHANGES_REQUESTED** [review]

File: dessia_common/workflow/blocks.py (hunk 3 above)
Comment (GhislainJ) [reply dessia_common/workflow/blocks.py:1403]: You're right ! For a lot of cases, it is old code legacy, but for the new ones, that is just me being careless. I tend to use `super().__init__()` more often, but sometimes I just slip into `ClassName.__init__()`.

I'll perform some changes

**Pull request merged**